Dockerfile-canary

```diff
@@ -42,6 +42,10 @@ IHBlcnNpc3RlbnQgc3RvcmFnZQogICAgICBjb3N0OiAkMC4wMAogICAgcGFyYW1ldGVyczogKl9w\
 CiAgICB1cGRhdGVzX3RvOgogICAgICAtIGRldgo="
 
 
+# install the client for use during bind
+RUN yum -y install postgresql && yum clean all
```

Comment:
What about the other 2 Dockerfiles? Maybe it could be added as a dependency to the postgresql-apb-role spec?